packages/utils/src/mergeSchemas.ts

```diff
@@ -32,7 +30,7 @@ export default function mergeSchemas(
       Array.isArray(right)
     ) {
       // Don't include duplicate values when merging 'required' fields.
-      acc[key] = union(left, right);
```

Comment:
I'm pretty sure this isn't a 1:1 as the `lodash` `union()` handles more use cases